src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,57 @@
+package tasks;
+
+import java.time.LocalDateTime;
+
+import common.DateTimeParser;
+
+public class Event extends Task {
+    private String from;
+    private String to;
+    private LocalDateTime fromDate;
+    private LocalDateTime toDate;
```

Comment:
Similar comment for 'from', 'to', 'fromDate', and 'toDate'! Would it be better if it was named more descriptively?